app/openai/mcp_client.py

```
@@ -0,0 +1,145 @@
+                yield session
+
+    async def get_pet_by_id(self, pet_id: int):
+        """ID로 펫 정보 조회"""
+        async with self.create_session() as session:
+            try:
+                result = await session.call_tool(
+                    "execute_sql",
+                    {"query": f"SELECT * FROM pet WHERE id = {pet_id}"}
+                )
+                return result.content[0].text if result.content else None
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)